src/qt/bitcoingui.cpp

```diff
@@ -396,6 +396,7 @@ void BitcoinGUI::createActions()
                     connect(activity, &OpenWalletActivity::opened, this, &BitcoinGUI::setCurrentWallet);
                     connect(activity, &OpenWalletActivity::finished, activity, &QObject::deleteLater);
                     connect(activity, &OpenWalletActivity::finished, dialog, &QObject::deleteLater);
+                    QMetaObject::invokeMethod(activity, "open");
```

Comment:
Note, removed the unnecessary explicit `Qt::QueuedConnection` argument. The default `Qt::AutoConnection` is enough since the activity thread is not the current thread.

Reply:
Nit: check the [return bool](https://doc.qt.io/qt-5/qmetaobject.html#invokeMethod), and maybe write a log message if this is false, or abort.

Reply:
Thanks, done.